MantidPlot/src/ProjectRecovery.cpp

```diff
@@ -490,7 +492,7 @@ void ProjectRecovery::saveWsHistories(const Poco::Path &historyDestFolder) {
   const auto &ads = Mantid::API::AnalysisDataService::Instance();
 
   // Hold a copy to the shared pointers so they do not get deleted under us
-  const auto wsHandles =
+  std::vector<boost::shared_ptr<Mantid::API::Workspace>> wsHandles =
```

Comment:
`const auto` was following the recommended practice post C++11. Is there some reason I've missed that this no longer works?